x-pack/legacy/plugins/spaces/server/lib/request_interceptors/on_post_auth_interceptor.ts

```diff
@@ -68,6 +68,7 @@ export function initSpacesOnPostAuthRequestInterceptor({
         const wrappedError = wrapError(error);
         return response.customError({
           body: wrappedError,
```

Comment:
Unrelated to this PR, should we be putting the full Boom error in the body? It looks like there's a test that is doing so here: https://github.com/elastic/kibana/blob/9a732014cbb05f36b631c88ac841f36285f54c49/src/core/server/http/integration_tests/router.test.ts#L823 

However, it feels like we should be using `wrappedError.body`. @restrry do you have any thoughts or suggestions on this?

Reply:
actually, it doesn't look like there's a `wrappedError.body` property. Were you thinking of `wrappedError.output.payload`?

`wrappedError.output.payload` looks something like this:
```
{ 
  statusCode: 401,
  error: 'Unauthorized',
  message: 'missing credendials',
  attributes: { error: 'missing credendials' } 
}
```

Reply:
Sorry for the conversation spam, but using `wrappedError.output.payload` actually generates a larger response than just using `wrappedError`:

Returning just `wrappedError` produces the following response:
```
{ 
  statusCode: 401,
  error: 'Unauthorized',
  message: 'missing credendials'
}
```

Whereas `wrappedError.output.payload` returns
```
{ 
  statusCode: 401,
  error: 'Unauthorized',
  message: 'missing credendials',
  attributes: { error: 'missing credendials' } 
}
```

Reply:
You're right, I was looking at the `wrappedError` in an inspector, and there are certain properties which exist there which aren't document or in the type definition. My initial concern was that the full Boom error has a [bunch of properties](https://github.com/hapijs/boom/blob/master/API.md#boom) which didn't seem like they should be included in the response body itself.

However, it looks like the error which we're passing into the `body` property ends up being passed into [h.response](https://github.com/hapijs/hapi/blob/master/API.md#-hresponsevalue), falling into the pit of undocumented Hapi behaviors, and is displayed somewhat reasonably.

So, I think leaving this alone is likely the best course of action.